chrome-extension/manifest.js

```diff
@@ -51,7 +51,7 @@ const manifest = deepmerge(
       },
       {
         matches: ['http://*/*', 'https://*/*', '<all_urls>'],
-        js: ['content-ui/index.iife.js'],
+        js: ['content-ui/page1/index_dev.js'],
```

Comment:
Because of missing implementation about shadow dom injection, this is totally not working right now

Reply:
erm bro.. basically what you need to do is just this :
```ts
import { createRoot } from 'react-dom/client';
import '@src/index.css';
import '../dist/global-out.css';
import NewTab from '@src/NewTab';
```